src/databricks/labs/ucx/workspace_access/generic.py

```diff
@@ -60,9 +60,12 @@ def __iter__(self):
 
 
 class GenericPermissionsSupport(AclSupport):
-    def __init__(self, ws: WorkspaceClient, listings: list[Listing]):
+    def __init__(
+        self, ws: WorkspaceClient, listings: list[Listing], verify_timeout: timedelta | None = timedelta(minutes=20)
```

Comment:
```suggestion
        self, ws: WorkspaceClient, listings: list[Listing], verify_timeout: timedelta | None = timedelta(minutes=1)
```
20 minutes is too much, 1 has to be enough